src/adapter.rs

```diff
@@ -75,6 +80,14 @@ impl Adapter {
     pub fn ip_addresses(&self) -> &[IpAddr] {
         &self.ip_addresses
     }
+    /// Get the adapter's prefixes
+    pub fn prefixes(&self) -> &Vec<(IpAddr, u32)> {
```

Comment:
Return `&[(IpAddr, u32)]` instead of `&Vec<(IpAddr, u32)>`

Reply:
Could you add some more docs here?
Like what an adapter prefix means, what the number in the tuple represents, and perhaps a doctest that shows how to use it (because somehow you have to cast `IpAddr` to bytes in order to use it (?)).